### PR TITLE
Test initial commit

### PR DESCRIPTION
**Pull request by adrianryt**

### CODE REVIEW

**Comment by wawel37** [comment]
LGTM!

**Review by wawel37 — APPROVED** [review]

File: src/main/logic/App.java [hunk 1]
```diff
@@ -1,5 +1,8 @@
+package main.logic;
+
 public class App {
     public static void main(String[] args) {
-        System.out.println("notarial-blockchain");
+
```
Comment (xsanm) [inline src/main/logic/App.java:5]: unused `\n`

**Review by xsanm — CHANGES_REQUESTED** [review]
Looks good overall, minor improvement will be needed

**Review by xsanm — APPROVED** [review]
LGTM

**Pull request merged**